libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32H757xx.py

```diff
@@ -54,10 +54,10 @@
     # we can't use DTCM first for main firmware as some builds overflow the first segment
     'RAM_MAP_BOOTLOADER' : [
         (0x20000000, 128, 2), # DTCM, tightly coupled, no DMA, fast
-        (0x30000000, 256, 0), # SRAM1, SRAM2
+        (0x30000000, 256, 8), # SRAM1, SRAM2
         (0x24000000, 512, 4), # AXI SRAM. Use this for SDMMC IDMA ops
         (0x00000400,  63, 2), # ITCM (first 1k removed, to keep address 0 unused)
-        (0x30040000,  32, 0), # SRAM3.
+		(0x30040000,  32, 8), # SRAM3.
```

Comment:
nitpick: little whitespace issue.  looks like tabs have been added instead of spaces.

Reply:
Feel free to use the `black` formatter which is used in pre-commit to automatically handle these changes.